src/internal/connector/support/m365Transform.go

```diff
@@ -308,6 +333,17 @@ func ToItemAttachment(orig models.Attachmentable) (models.Attachmentable, error)
 
 		transform.SetItem(newEvent)
 
+		return transform, nil
+	case mailIA:
+		message := item.(models.Messageable)
+
+		newMessage, err := sanitizeMessage(message)
+		if err != nil {
+			return nil, err
+		}
+
+		transform.SetItem(newMessage)
+
 		return transform, nil
 	default:
 		return nil, fmt.Errorf("exiting ToItemAttachment: %s not supported", *itemType)
```

Comment:
How is this error handled - does restore fail or do we mask the error?

Reply:
This is logged at the info level here:
https://github.com/alcionai/corso/blob/27c1e5c51140b238c6c3cbc3df79b290e37704fe/src/internal/connector/exchange/attachment.go#L72

Reply:
Perfect - could you please tweak that message - let's make it `item attachment restore not supported for this type. skipping upload.` . 

Do we have a test case that exercises this? Since it's in the error case - I want to make sure we're not accessing any fields in the logging that would panic.